components/match2/commons/match_summary_ffa.lua

```diff
@@ -570,7 +570,26 @@ function MatchSummaryFfa.updateMatchOpponents(match)
 	Array.forEach(match.opponents, function(opponent, idx)
 		opponent.placementStatus = match.extradata.status[idx]
 	end)
+end
+
+---@param game table
+---@param matchOpponents table[]
+function MatchSummaryFfa.updateGameOpponents(game, matchOpponents)
+	-- Add match opponent data to game opponent
+	game.opponents = Array.map(game.opponents,
+		function(gameOpponent, opponentIdx)
+			local matchOpponent = matchOpponents[opponentIdx]
+			local newGameOpponent = Table.merge(matchOpponent, gameOpponent)
+			-- These values are only allowed to come from Game and not Match
+			newGameOpponent.placement = gameOpponent.placement
+			newGameOpponent.score = gameOpponent.score
+			newGameOpponent.status = gameOpponent.status
+			return newGameOpponent
+		end
+	)
 
+	-- Sort game level based on placement
+	Array.sortInPlaceBy(game.opponents, FnUtil.identity, MatchSummaryFfa.placementSortFunction)
```

Comment:
suggest to adjust the `placementSortFunction` a tiny bit:
in the last line of that function:
sort by lowercased opponent name
from: `return (opponent1.name or '') < (opponent2.name or '')`
to: `return (opponent1.name or ''):lower() < (opponent2.name or ''):lower()`